ui/src/components/ImageViewer/ImageViewer.jsx

```diff
@@ -4,8 +4,16 @@ import GalleryModal from './GalleryModal';
 import styles from './imageViewer.module.css';
 
 export default function ImageViewer(props) {
-  const { galleryView, imgUrls, imgAlt, imgTargetX, imgTargetY, imageName } =
-    props;
+  const {
+    galleryView,
+    imagegUrls,
+    imagegUrl,
```

Comment:
Typos?


```suggestion
    imageUrls,
    imageUrl,
```

Multiple occurrences in this pull request.